rubicon_ml/client/project.py

```diff
@@ -289,10 +290,17 @@ def experiment(self, id=None, name=None):
                 )
 
             experiment = experiments[-1]
```

Comment:
lines 281-293 need to be inside your try/except as well. this first block is for getting an experiment by name - right now, only retrieval by ID will work with the mult-backend. this whole `if` should be inside the `try`. then there should also be an `else` in the `try` that has the current `try` code